src/GradhSph/GradhSph.cpp

```diff
@@ -207,7 +207,7 @@ int GradhSph<ndim, kernelclass>::ComputeH
 
     // Density must at least equal its self-contribution
     // (failure could indicate neighbour list problem)
-    assert(parti.rho >= parti.m*parti.hfactor*kern.w0_s2(0.0));
+    assert(parti.rho >= 0.99*parti.m*parti.hfactor*kern.w0_s2(0.0));
```

Comment:
I've had some bug-reports regarding this line when the computed density is exactly equal to the self-contribution.  It suggests some other problem (think it's to do with the sinks which I've solved in another branch) but also it could be the guess is simply too small so there's only the self-contribution, so I think making this slightly smaller than 1 is fine to stop this assert killing the code when there's not a problem.